driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -80,6 +87,31 @@ public UnifiedTest(final String schemaVersion, @Nullable final BsonArray runOnRe
         this.context.push(ContextElement.ofTest(definition));
     }
 
+    @NotNull
+    protected static Collection<Object[]> getTestData(final String directory) throws URISyntaxException, IOException {
```

Comment:
Refactored test data loading now that there are several different unified test runners